src/background/services/identity.ts

```diff
@@ -35,27 +35,32 @@ export default class IdentityService {
   }
 
   public unlock = async (): Promise<boolean> => {
-    await this.setDefaultIdentity(true);
+    await this.setDefaultIdentity();
 
     return true;
   };
 
-  public setActiveIdentity = async ({
+  public setActiveIdentity = async ({ identityCommitment }: { identityCommitment: string }): Promise<boolean> => {
```

Comment:
I dont see it make sense actually to have this function `setActiveIdentity`, it is redundant in this context. 
It's useless because it is just used to pass `const identities = await this.getIdentitiesFromStore();` to `updateActiveIdentity()` where as you can simply get `identities` directly inside `updateActiveIdentity()` withtout the need to pass it.

Reply:
The idea here is: `setActiveIdentity` is public method and `updateActiveIdentity` is private one.
For public: we need to get identities from store to check if we have active identity.
For private: we already have identities list so we just need to set active identity. In addition it fixes connection error so I can remove `updateUI` flag and keep the same performance.